src/solvers/dgsem_tree/dg.jl

```diff
@@ -43,8 +43,8 @@ function volume_jacobian(element, mesh::TreeMesh, cache)
 end
 
 @inline function get_inverse_jacobian(inverse_jacobian, mesh::TreeMesh,
-                                      element, indices...)
-    return inverse_jacobian[element]
+                                      indices...)
+    return inverse_jacobian[last(indices...)]
```

Comment:
```suggestion
                                      indices...)
    element = last(indices...)
    return inverse_jacobian[element]
```
Maybe for clarity?

Reply:
Sure, that works for me. Although I used `last(indices)` instead because `last(indices...)` threw the following error
```julia
julia> Trixi.get_inverse_jacobian(semi.cache.elements.inverse_jacobian, mesh, 2,2,6)
ERROR: MethodError: no method matching last(::Int64, ::Int64, ::Int64)

Closest candidates are:
  last(::Any, ::Integer)
   @ Base abstractarray.jl:549
  last(::Number)
   @ Base number.jl:112
  last(::AbstractString, ::Integer)
   @ Base strings/basic.jl:712
```

Reply:
Sorry, that was an error on my side. Thanks